spec/requests/items_requests_spec.rb

```diff
@@ -134,6 +135,39 @@
       end
     end
 
+    describe "with invalid parameters" do
+      let(:invalid_item_params) do
+        {
+          item: {
+            name: "",  # Invalid name
+            partner_key: create(:base_item).partner_key,
+            value_in_cents: -100,  # Invalid value
+            package_size: nil,
+            distribution_quantity: nil
+          }
+        }
+      end
+  
+      it "loads and displays the item categories when rendering new" do
+        # Create some item categories for the organization
+        item_categories = create_list(:item_category, 3, organization: organization)
+  
+        # Attempt to create an item with invalid parameters
+        post items_path, params: invalid_item_params
+  
+        # Expect to render the new template
+        expect(response).to render_template(:new)
+  
+        # Ensure the item categories are assigned in the controller
+        expect(assigns(:item_categories)).to eq(organization.item_categories.order('name ASC'))
```

Comment:
I much prefer giving the categories explicit values and testing against those values rather than pulling them from the DB. You end up basically just copying the code you're testing into the test.

Reply:
Sorry, I'm having trouble understanding exactly what you mean here.

Reply:
I mean doing something like

```ruby
let(:categories) do 
  [
    FactoryBot.create(:item_category, name: 'Apples'),  # include whatever other IDs need to be passed in
    FactoryBot.create(:item_category, name: 'Bananas')
  ] 
end
# ...
expect(assigns(:item_categories)).to eq(categories))
```

Reply:
Hey @bdeanhardt -- do you need further guidance on this?

Reply:
Hey @cielf! Past 2 weeks have been chaos, but I'm back! I'm taking a look today :)

Reply:
So sorry about the delay! How does it look now?

Reply:
You shouldn't need to specify an `order` in the spec. 

To be a bit more directive, here's what I had in mind:

```ruby
let!(:category1) { FactoryBot.create(:item_category, name: 'Bananas', organization: organization) }
let!(:category2) { FactoryBot.create(:item_category, name: 'Apples' organization: organization) }
let!(:category3) { FactoryBot.create(:item_category, name: 'Umbrella', organization: organization) }

# ...

expect(assigns(:item_categories)).to eq([category2, category1, category3])
```